### PR TITLE
fix: airgapped instance bug fixes

### LINKED ISSUE #22361: [Bug]: The application is not loading in the air gaped instance
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

The application is not loading in the air gaped instance
image

### Steps To Reproduce

1. first pull the image docker pull ankitakinger/appsmith:airgap
2. now switch off the internet 
3. now docker run -e APPSMITH_AIRGAP_ENABLED=true -e KEYGEN_LICENSE_VERIFICATION_KEY=b2b98c313548c14cb22ddb4afb9124553c4c829c290d8a3cbf01ff8ee57624dc -d --name appsmith-airgap -p 80:80 -v "$PWD/stacks:/appsmith-stacks" ankitakinger/appsmith:airgap
4. Now do the set up 
5. add air gapped license key
6. go to home page click on any app 
7. The app loads in a infinite loop

### Public Sample App

_No response_

### Environment

Release

### Issue video log

_No response_

### Version

release

### LINKED ISSUE #22375: [Bug]: In firefox browser the logo seems to be broken in the branding page
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

In firefox browser the logo seems to be broken in the branding page
image

### Steps To Reproduce

1. first pull the image docker pull ankitakinger/appsmith:airgap
2. now switch off the internet
3. now docker run -e APPSMITH_AIRGAP_ENABLED=true -e KEYGEN_LICENSE_VERIFICATION_KEY=b2b98c313548c14cb22ddb4afb9124553c4c829c290d8a3cbf01ff8ee57624dc -d --name appsmith-airgap -p 80:80 -v "$PWD/stacks:/appsmith-stacks" ankitakinger/appsmith:airgap
4. Now do the set up
5. add air gapped license key
6. go to home page 
7. click on admin setting
8. click on branding
9. The image seems to be broken in the last card

### Public Sample App

_No response_

### Environment

Release

### Issue video log

_No response_

### Version

release

### LINKED ISSUE #22392: [Bug]: Air Gapped instance - Generate page with Crud - Success message image missing
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

In an airgapped intstance, when I try to generate data with crud, the success message image is missing

### Steps To Reproduce

1. Pull the image with the command docker pull ankitakinger/appsmith:airgap
2. Switch off internet and execute the following:
docker run -e APPSMITH_AIRGAP_ENABLED=true -e KEYGEN_LICENSE_VERIFICATION_KEY=b2b98c313548c14cb22ddb4afb9124553c4c829c290d8a3cbf01ff8ee57624dc -d --name appsmith-airgap -p 80:80 -v "$PWD/stacks:/appsmith-stacks" ankitakinger/appsmith:airgap
3. Now do the set up
4. Add air gapped license key
5. Create a new DS and try and generate page with data
6. Note that the success message does not have the image.

Crud_Success Airgapped

The image on release is as below: 

Crud_Success

### Public Sample App

_No response_

### Environment

Release

### Issue video log

_No response_

### Version

Self Hosted

### LINKED ISSUE #22394: [Bug]: The icons are broken for authenticated API and authenticated GraphQL
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

The icons are broken for authenticated API and authenticated GraphQL
image

### Steps To Reproduce

1. first pull the image docker pull ankitakinger/appsmith:airgap
2. now switch off the internet
3. now docker run -e APPSMITH_AIRGAP_ENABLED=true -e KEYGEN_LICENSE_VERIFICATION_KEY=b2b98c313548c14cb22ddb4afb9124553c4c829c290d8a3cbf01ff8ee57624dc -d --name appsmith-airgap -p 80:80 -v "$PWD/stacks:/appsmith-stacks" ankitakinger/appsmith:airgap
4. Now do the set up
5. add air gapped license key
6. go to home page
7. click on edit 
8. click on + icon next to datasource
9. The icons are broken for authenticated API and authenticated GraphQL

### Public Sample App

_No response_

### Environment

Release

### Issue video log

_No response_

### Version

release

### LINKED ISSUE #22395: [Bug]: Some unknow widgets are displayed in widgets section
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Some unknow widgets are displayed in widgets section
image

### Steps To Reproduce

1. first pull the image docker pull ankitakinger/appsmith:airgap
2. now switch off the internet
3. now docker run -e APPSMITH_AIRGAP_ENABLED=true -e KEYGEN_LICENSE_VERIFICATION_KEY=b2b98c313548c14cb22ddb4afb9124553c4c829c290d8a3cbf01ff8ee57624dc -d --name appsmith-airgap -p 80:80 -v "$PWD/stacks:/appsmith-stacks" ankitakinger/appsmith:airgap
4. Now do the set up
5. add air gapped license key
6. go to home page
7. 

### Public Sample App

_No response_

### Environment

Release

### Issue video log

_No response_

### Version

release

### LINKED ISSUE #22437: [Bug]: In the file picker overlay, No internet connection error is showing up
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

-

### Steps To Reproduce

Screenshot 2023-04-14 at 2 59 14 PM

### Public Sample App

_No response_

### Environment

Deploy Preview

### Issue video log

_No response_

### Version

Release

### LINKED ISSUE #22441: [Bug]: Air Gapped instance - Two icon widgets are displayed on the canvas
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

In an airgap instance, when the user searches for the icon widget in the search bar then 2 icon widgets are displayed.

### Steps To Reproduce

1. Pull the image with the command docker ```pull docker pull appsmith/appsmith-airgap:test```
2. Switch off internet and execute the docker run command
3. Now do the set up
4. Add air gapped license key.
5. Inside the application, in the widget search bar search for **Icon** widget & observe
 **Note**
Please refer the attached SS:- 
image

### Public Sample App

_No response_

### Environment

Release

### Issue video log

_No response_

### Version

Self Hosted

### PR DESCRIPTION
**Pull request by berzerkeer**
## Description

- This PR fixes few of the bugs on airgapped instances.

Fixes #22361 
Fixes #22375 
Fixes #22392 
Fixes #22394 
Fixes #22395 
Fixes #22441 
Fixes #22437 

Media
> A video or a GIF is preferred. when using Loom, don’t embed because it looks like it’s a GIF. instead, just link to the video

## Type of change

- Bug fixes (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Manual

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/src/pages/Settings/FormGroup/ImageInput.tsx [hunk 1]
```diff
@@ -59,7 +60,11 @@ export const ImageInput = (props: ImageInputProps) => {
         className ? className : ""
       }`}
     >
-      <img alt="Preview" className="h-8" src={preview || value} />
+      <img
+        alt="Preview"
+        className="h-8"
+        src={getAssetUrl(preview as any) || getAssetUrl(value)}
```
Comment (ankitakinger) [inline app/client/src/pages/Settings/FormGroup/ImageInput.tsx:66]: does this not work this way `getAssetUrl(preview || value)`?

**Review by ankitakinger — DISMISSED** [review]

**Review by dipyamanbiswas07 — APPROVED** [review]

**Review by ankitsrivas14 — APPROVED** [review]

**Pull request merged**